src/components/Label/tests/Label.test.tsx

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+import {mount} from 'enzyme';
+import Label from '../Label';
+
+describe('Label Suit', () => {
+
+    it('Verify Label is not Hidden', () => {
```

Comment:
I would make it a complete sentence for readability.

it('should verify Label is not hidden', () => {